### ISSUE TITLE
Missing canned Whitelist

### ISSUE THREAD

**Issue by droidery** [scinfu/SwiftSoup#6]
I tried using your code sample in my project.

```
do{
    let unsafe: String = "<p><a href='http://example.com/' onclick='stealCookies()'>Link</a></p>"
    let safe: String = try SwiftSoup.clean(unsafe, Whitelist.basic())!
    // now: <p><a href="http://example.com/" rel="nofollow">Link</a></p>
}catch Exception.Error(let type, let message){
    print(message)
}catch{
    print("error")
}
```
There's an error: `Type 'Whitelist' has no member 'basic'`. `SwiftSoup.Whitelist.basic()` also isn't defined.

Looks like there should be `Whitelist` with some pre-canned rules: https://jsoup.org/apidocs/org/jsoup/safety/Whitelist.html

**Comment by scinfu** [comment]
Update to latest version , WhiteList is part of SwiftSoup , maybe you have an old version.
I tested your code with `1.1.6` version and work .

**Comment by scinfu** [comment]
@droidery It's available 1.1.7 with some fixes in WhiteList , i suggest you to update the library .

**Comment by droidery** [comment]
Thank you, I will do that.
